docs/use/connect-your-wallet/keplr.mdx

```diff
@@ -6,6 +6,17 @@ sidebar_position: 2
 
 Connect your Keplr wallet with Evmos.
 
+import { chainConfig, handler } from '@site/src/utils/ChainConfig';
```

Comment:
would remove this here, as new users will want to connect to **Mainnet**

Reply:
Not sure I agree but I moved it to the testnet of this wallet section